webclient/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line
+import React, { useMemo, useState } from 'react';
+
+import { CardDTO } from 'services';
+
+import './Card.css';
+
+interface CardProps {
+  card: CardDTO;
+}
+
+const Card = ({ card }: CardProps) => {
+  const src = `https://api.scryfall.com/cards/multiverse/${card?.identifiers?.multiverseId}?format=image`;
```

Comment:
Should use ScryfallID imo as gatherer is not complete. Also, should give user an option, in the future, to pick their download provider